python/tank/platform/constants.py

```diff
@@ -114,6 +114,16 @@
 # the file to look for that defines and bootstraps an app
 APP_FILE = "app.py"
 
+# an optional stylesheet that can be defined by bundles
+BUNDLE_STYLESHEET_FILE = "style.qss"
+
+# define the standard "shotgun blue"
+SG_HIGHLIGHT_COLOR = "#30A7E3"
+
+# define the standard stylesheet replacements 
+QSS_REPLACEMENTS = { "SG_HIGHLIGHT_COLOR": SG_HIGHLIGHT_COLOR,
+                     "SG_ALERT_COLOR": "#FC6246"}
```

Comment:
Feels weird to have the dictionary with one entry being its own constant and the other a string.
Would it make more sense / be easier to extend if the global was named more like:
SG_STYLESHEET_CONSTANTS and then below rather than using the SG_HIGHLIGHT_COLOR constant the reference was into this dict?  So something like

```python
SG_STYLESHEET_CONSTANTS["SG_HIGHLIGHT_COLOR"]
```
?